src/tracker.md

```diff
@@ -17,16 +17,16 @@ When pressing a button, the icon above it will change. The background of the app
 
 The `tracker::track` macro implements the following methods for your struct fields:
 
-+ `get_#field_name()`  
++ `get_#field_name()`
   Get an immutable reference to your field.
 
-+ `get_mut_#field_name()`  
++ `get_mut_#field_name()`
   Get a mutable reference to your field. Assumes the field will be modified and marks it as changed.
 
-+ `set_#field_name(value)`  
++ `set_#field_name(value)`
   Get a mutable reference to your field. Marks the field as changed only if the new value isn't equal with the previous value.
 
-+ `update_#field_name(fn)`  
++ `update_#field_name(fn)`
```

Comment:
I think the spaces at the end of those lines were there on purpose to add a line break.

Reply:
reverted.